src/server/middleware/blob/put_blob_upload.go

```diff
@@ -63,4 +101,8 @@ func PutBlobUploadMiddleware() func(http.Handler) http.Handler {
 
 		return nil
 	})
+
+	return func(next http.Handler) http.Handler {
+		return alice.New(before, after).Then(next)
+	}
```

Comment:
There is a function named `Chain` in the middleware pkg which does the same thing as `alice.New`.

Reply:
updated.